api-report/container-loader.api.md

```diff
@@ -183,7 +183,7 @@ export class DeltaManager extends TypedEventEmitter<IDeltaManagerInternalEvents>
     // (undocumented)
     get disposed(): boolean;
     // (undocumented)
-    emitDelayInfo(id: string, delayMs: number, error: ICriticalContainerError): void;
+    emitDelayInfo(id: string, delayMs: number, error: any): void;
```

Comment:
I think you can add this in Breaking.md too as it as public api?

Reply:
could we go with unknown instead of any? that will force the function to deal with typing before using

Reply:
Sure.  I'll keep that in mind on future PRs too, most places should probably do `unknown` so we can isolate the places that actually inspect them.